dotcom-rendering/src/server/lib/logging.ts

```diff
@@ -74,10 +74,14 @@ const enableLog4j = {
 			// Owner Read & Write, Group Read
 			mode: 0o640,
 		},
+		out: {
+			type: 'stdout',
+			layout: { type: 'json', separator: ',' },
```

Comment:
Are the logs expecting JSON?

Reply:
Yep, @mxdvl!  The DevX logs default parser is JSON. You can find the config [here](https://github.com/guardian/devx-logs/blob/d55a126f5e0519993b915d3416985b20147ff7d6/ec2/fluentbit/application-logs.conf#L21C4-L21C4) if you're interested.